lib/process-services/task-list/components/start-task.component.html

```diff
@@ -65,7 +65,7 @@
                 </div>
 
                 <div class="adf-grid-column adf-grid-half-width">
-                    <adf-people-selector [(peopleId)]="assigneeId" id="assignee_id"></adf-people-selector>
+                    <people-widget [(peopleId)]="assigneeId" [field]="field"></people-widget>
```

Comment:
is the people widget able to show the person?
I think we are only using the output and not the two way binding
If so we can change this code

 ```@Output()
+    peopleIdChange: EventEmitter<number>;```
to
 ```@Output()
+    peopleSelected: EventEmitter<number>;```